core/tests/Test.Microsoft.Identity.LabInfrastructure/LabServiceApi.cs

```diff
@@ -68,6 +68,11 @@ private IEnumerable<IUser> GetUsersFromLab(UserQueryParameters query)
             //Fetch user
             string result = webClient.DownloadString("http://api.msidlab.com/api/user");
 
+            if (String.IsNullOrWhiteSpace(result))
+            {
+                throw new Exception("No lab user with specified parameters exists");
```

Comment:
>throw new Exception [](start = 16, length = 19)

nit: you should create a specific exception for this

Reply:
InvalidOperationException is the "common" exception type when you don't want to have a new exception (.e.g. only throw it once). Probably worth having a specific one for these tests.